screeps-game-api/src/objects/macros.rs

```diff
@@ -37,52 +64,31 @@ macro_rules! simple_accessors {
                 }
             )*
         }
-    );
-    (
```

Comment:
Good catch on this. I think these must have been from before all the traits had default implementations of getter methods.